recipe/build.sh

```diff
@@ -18,6 +18,15 @@ sed -i 's@overload_caster_t@override_caster_t@g' plugins/python/src/*.cpp
 rm -rf plugins/python/include/pybind11
 ln -s $PREFIX/include/pybind11 $PWD/plugins/python/include/pybind11
 
+# Make the compilers look for crypt.h in the conda include directory
+# for Python 3.8 builds
+# c.f. https://github.com/conda-forge/linux-sysroot-feedstock/issues/52
+if [[ "${target_platform}" == linux-* ]]; then
+    if [[ $(python -c "import sys; print(sys.version_info[:2] < (3, 9))") == "True" ]]; then
+        export CPATH="${PREFIX}/include:${CPATH}"
+    fi
+fi
```

Comment:
@jakirkham as I am very much counting myself amount the people who were confused and "linking to issue ( conda-forge/linux-sysroot-feedstock#52 ) with various hacks they had to implement." https://github.com/conda-forge/python-feedstock/pull/668/files#r1525368066 (c.f. my comment in the PR body) &mdash; reading https://github.com/conda-forge/python-feedstock/pull/668 was actually informative only because of the links you posted, so thank you! &mdash; can you comment on if there is a more preferred method of updating paths (as was done in https://github.com/conda-forge/torchmd-net-feedstock/pull/21) or `cp`ing files into the required location (as was done in https://github.com/conda-forge/ray-packages-feedstock/pull/150)? I suspect the answer is "it doesn't matter as this is happening at build time and so whatever you do there that works is fine" but I thought I would ask to avoid potentially doing something that seems fine but isn't a great pattern in general.